src/patito/validators.py

```diff
@@ -175,6 +175,8 @@ def _find_errors(  # noqa: C901
             .select(column)
             # Remove those rows that do not contain lists at all
             .filter(pl.col(column).is_not_null())
+            # Remove empty lists
+            .filter(pl.col(column).list.len() > 0)
             # Convert lists of N items to N individual rows
             .explode(column)
```

Comment:
Ah [`polars.DataFrame.explode`](https://docs.pola.rs/api/python/dev/reference/dataframe/api/polars.DataFrame.explode.html) keeps empty lists and nulls (converting both to null in the exploded column. This surprised me since `pyspark.sql.functions.explode` drops these values with a separate `explode_outer` function that keeps them. So `polars.DataFrame.explode` is this second type as documented by pyspark:

> Unlike explode, if the array/map is null or empty then null is produced.

```py
import polars as pl
df = pl.DataFrame(
    {
        "letters": ["a", "b", "c"],
        "numbers": [[], [4, 5], None],
    }
)
df.explode("numbers")
```

Thanks for the fix!